apps/omg_child_chain/config/config.exs

```diff
@@ -18,4 +18,6 @@ config :omg_child_chain, OMG.ChildChain.Tracer,
   disabled?: true,
   type: :omg_child_chain
 
+config :statix, OMG.ChildChain.Application, prefix: "omg"
```

Comment:
hm... I think this isn't needed?

Reply:
@InoMurko 

It's a subjective topic. but all integrations we use have a name prefix, vmstats, docker, cloudflare, system, pingdom, trace..

i'd recommend having an `omg.` prefix for our business-related metrics, bare metrics can cause namespace clashes with other integrations that are buggy or not prefixed, etc, etc...many examples where it could go wrong.

if you prefer, i will take it out. just say the word.